modules/services/localization/src/main/res/values/strings.xml

```diff
@@ -2133,6 +2133,8 @@
 
     <!-- Referrals -->
     <string name="gift">Gift</string>
+    <string name="referrals_claim_guess_pass_banner_card_title">Claim your 2-Month Guest Pass to Plus</string>
```

Comment:
I wonder if it would look nicer if we moved the Guest Pass to Plus on the next line, similar to the design. We could do it by adding non-breaking spaces. 

<img width="290" alt="Frame 1000004472" src="https://github.com/user-attachments/assets/4f2e9775-3c58-4db4-93ed-f9f5edabfd25">

Reply:
Changed for only english lang in 615ba98

<img width=400 src="https://github.com/user-attachments/assets/6c437ac5-9b1e-4e7d-ada4-47edbc019d91"/>